models/nlp/albert/run_pretraining.py

```diff
@@ -64,11 +64,13 @@
 )
 
 # See https://github.com/huggingface/transformers/issues/3782; this import must come last
-import smdistributed.dataparallel.tensorflow as herring  # isort:skip
-import herringcommon as hc
+import smdistributed.dataparallel.tensorflow as smddp  # isort:skip
+
+import smddpcommon
```

Comment:
shouldn't we refactor to not use `smddpcommon`? We don't expose that API to public.

Reply:
no you have refactored already and there is not herringcommon api in our release, but this thing is needed for setting up the bucket size

Reply:
Yes ofcourse herringcommon is not to be used. I refactored it :)
Yes I know it is being used to set bucket size. 

Rephrasing my above question; shouldn't we "rewrite" above logic to NOT use smddpcommon. Speak: rewrite the logic that sets bucket size using gradient tape rather than use smddpcommon. Because we dont expose smddpcommon API to public